web/README.md

```diff
@@ -55,16 +62,17 @@ Having initialised secret storage, then add the following section to `secrets.js
   }
 ```
 
-### Build front-end library
-To use the GOV.UK Design System and front-end components locally then:
+### Build the front-end library
+To use the GOV.UK Design System and front-end components locally:
 - Navigate to the root of the Web APP `.\web\src\Web.App`
 - Install the required packages `npm i`
 - Run the gulp script to build ssas and copy assets `npm run-script build`
 
 ### Running tests
-
+[comment]: <> (Does this statement need to be expanded to include something around the user needing to manually run the unit tests in the next section?)
```

Comment:
Running test locally should always be done. The statement is more to raise aware of the automation.

Reply:
Is it worth specifying it to be on the safe side? We can't anticipate which developers might pick this up at a later date, and they may not do this.

Reply:
It's developer best practice - I don't think we should be explicitly stating does and don'ts

Reply:
It a developer throws up a PR without running tests first then it's their risk.